node/rust-client/src/lib.rs

```diff
@@ -137,6 +164,70 @@ pub fn start_socket_listener_external(env: Env) -> Result<JsObject> {
     Ok(promise)
 }
 
+fn ensure_tokio_runtime() -> &'static Runtime {
+    static TOKIO: OnceLock<Runtime> = OnceLock::new();
+    TOKIO.get_or_init(|| Runtime::new().expect("Failed to create Tokio runtime"))
+}
+
+#[napi(js_name = "InitOpenTelemetry")]
+pub fn init_open_telemetry(open_telemetry_config: OpenTelemetryConfig) -> Result<()> {
+    let mut config = GlideOpenTelemetryConfigBuilder::default();
+    // initilaize open telemetry traces exporter
+    if let Some(traces) = open_telemetry_config.traces {
+        if let Some(endpoint_str) = traces.endpoint {
+            config = config.with_trace_exporter(Some(
+                GlideOpenTelemetrySignalsExporter::from_str(&endpoint_str)
+                    .map_err(ConnectionError::IoError)
+                    .map_err(|e| napi::Error::new(Status::Unknown, format!("{}", e)))?,
+            ));
+        }
+        if let Some(requests_percentage) = traces.requests_percentage {
+            config = config.with_trace_requests_precentage(requests_percentage);
+        }
+    }
+
+    if let Some(metrics) = open_telemetry_config.metrics {
+        if let Some(endpoint_str) = metrics.endpoint {
+            config = config.with_metrics_exporter(Some(
+                GlideOpenTelemetrySignalsExporter::from_str(&endpoint_str)
+                    .map_err(ConnectionError::IoError)
+                    .map_err(|e| napi::Error::new(Status::Unknown, format!("{}", e)))?,
+            ));
+        }
+    }
+
+    let flush_interval_ms = open_telemetry_config
+        .flush_interval_ms
+        .unwrap_or(DEFAULT_FLUSH_SIGNAL_INTERVAL_MS as i64);
+
+    if flush_interval_ms <= 0 {
+        return Err(napi::Error::new(
+            Status::Unknown,
+            "InvalidInput: flushIntervalMs must be a positive integer".to_owned(),
+        ));
+    }
+
+    config = config.with_flush_interval(std::time::Duration::from_millis(flush_interval_ms as u64));
+
+    // Initialize OpenTelemetry synchronously
+    ensure_tokio_runtime().block_on(async {
```

Comment:
I don't understand why do we need to run it on tokio runtime? it isn't an async task to init GlideOpenTelemetry, right? so why do we need to create a tokio runtime and wrap it as async and run it in?
can't we remove all of this part and simply do-
```
        if let Err(e) = GlideOpenTelemetry::initialise(config.build()) {
            log(
                Level::Error,
                "OpenTelemetry".to_string(),
                format!("Failed to initialize OpenTelemetry: {}", e),
            );
            return Err(napi::Error::new(
                Status::Unknown,
                format!("Failed to initialize OpenTelemetry: {}", e),
            ));
        }